Modules/Core/Common/src/itkMultipleLogOutput.cxx

```diff
@@ -20,15 +20,9 @@
 
 namespace itk
 {
-MultipleLogOutput::MultipleLogOutput()
-{
-  this->m_Output.clear();
```

Comment:
I assume this was a no-op?

Reply:
Indeed! m_Output is an `std::set`, so its default-constructor already creates an empty container.